packages/charts/react-charting/src/components/DeclarativeChart/helpers.ts

```diff
@@ -0,0 +1,280 @@
+import { create as d3Create, select as d3Select, Selection } from 'd3-selection';
```

Comment:
this helper is related to generate image. You can rename it to imageExporter.ts

Reply:
done